src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -67,7 +100,9 @@
     private static final int    NUM_MOBS_ACHIEVEMENT_THIRD  = 100;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "NUM_MOBS_ACHIEVEMENT_THIRD" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)